src/libraries/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -209,13 +207,11 @@ internal static bool TryReadMemoryValueFromFile(string path, out ulong result)
         private static unsafe CGroupVersion FindCGroupVersion()
         {
             CGroupVersion cgroupVersion = CGroupVersion.None;
-            const int MountPointFormatBufferSizeInBytes = 32;
-            byte* formatBuffer = stackalloc byte[MountPointFormatBufferSizeInBytes];    // format names should be small
-            long numericFormat;
-            int result = Interop.Sys.GetFormatInfoForMountPoint(SysFsCgroupFileSystemPath, formatBuffer, MountPointFormatBufferSizeInBytes, &numericFormat);
-            if (result == 0)
+            Span<char> buffer = stackalloc char[16];
```

Comment:
This file was also parsing mountinfo, so I've updated it to share the parsing logic with DriveInfo.